tools/shared.py

```diff
@@ -2336,14 +2336,10 @@ def check_call(cmd, *args, **kw):
     raise
 
 def suffix(name):
-  parts = name.split('.')
-  if len(parts) > 1:
-    return parts[-1]
-  else:
-    return None
+  return os.path.splitext(name)[1]
```

Comment:
python docs say that splitexit returns a tuple with the second part containing the suffix *including the dot*. that changes the behavior here. how about returning `[1:]`?